gp2gp-translator/src/nonFunctionalTest/TestPlans/PS_NF_REL_2/PS_NF_REL_2.jmx

```diff
@@ -0,0 +1,295 @@
+            &quot;part&quot;: [&#xd;
+                {&#xd;
+                    &quot;name&quot;: &quot;includeSensitiveInfomation&quot;,&#xd;
+                    &quot;valueBoolean&quot;: true&#xd;
+                }&#xd;
+            ]&#xd;
+        }&#xd;
+    ]&#xd;
+}</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+              </elementProp>
+            </collectionProp>
+          </elementProp>
+          <stringProp name="HTTPSampler.domain">localhost</stringProp>
+          <stringProp name="HTTPSampler.port">${__P(facadePort, 8081)}</stringProp>
+          <stringProp name="HTTPSampler.protocol">http</stringProp>
+          <stringProp name="HTTPSampler.contentEncoding"></stringProp>
+          <stringProp name="HTTPSampler.path">/Patient/$gpc.migratestructuredrecord</stringProp>
+          <stringProp name="HTTPSampler.method">POST</stringProp>
+          <boolProp name="HTTPSampler.follow_redirects">true</boolProp>
+          <boolProp name="HTTPSampler.auto_redirects">false</boolProp>
+          <boolProp name="HTTPSampler.use_keepalive">true</boolProp>
+          <boolProp name="HTTPSampler.DO_MULTIPART_POST">false</boolProp>
+          <stringProp name="HTTPSampler.embedded_url_re"></stringProp>
+          <stringProp name="HTTPSampler.connect_timeout">120000</stringProp>
+          <stringProp name="HTTPSampler.response_timeout">120000</stringProp>
+          <stringProp name="TestPlan.comments">Send the initial EHR request with previously created variables and headers</stringProp>
+        </HTTPSamplerProxy>
+        <hashTree>
+          <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="${threadDetails} Response Assertion" enabled="true">
+            <collectionProp name="Asserion.test_strings">
+              <stringProp name="49588">202</stringProp>
+            </collectionProp>
+            <stringProp name="Assertion.custom_message">&quot;Failed to send initial response, or server did not return 202 (Accepted)&quot;</stringProp>
+            <stringProp name="Assertion.test_field">Assertion.response_code</stringProp>
+            <boolProp name="Assertion.assume_success">false</boolProp>
+            <intProp name="Assertion.test_type">8</intProp>
+            <stringProp name="TestPlan.comments">Assert that the response is the expected 202 (Accepted)</stringProp>
+          </ResponseAssertion>
+          <hashTree/>
+        </hashTree>
+        <PublisherSampler guiclass="JMSPublisherGui" testclass="PublisherSampler" testname="${threadDetails} Send RCMR_IN30000UK06 message" enabled="true">
+          <stringProp name="jms.jndi_properties">false</stringProp>
+          <stringProp name="jms.initial_context_factory">org.apache.qpid.jms.jndi.JmsInitialContextFactory</stringProp>
+          <stringProp name="jms.provider_url">/Users/martinwheeler/Documents/Configuration/jndi.properties</stringProp>
```

Comment:
Same as previous comment

Reply:
These were already in config.properties but this was not reflected in the jmx file.  Updated all occurrences of local references